### DISCUSSION TITLE
爱发电无法访问，希望官方能够考虑使用其他方式发布最新版本

### DISCUSSION THREAD

**Discussion by AkarinLiu** [Meloong-Git/PCL#4324] [讨论]
最近，爱发电无法正常访问，希望作者能够考虑在 GitHub 或 Gitee 发布二进制运行文件（应用程序），并且使用 NSIS 打包发布安装包。

**Comment by AkarinLiu** [comment]
作者会考虑使用 GitHub Pages 搭建 PCL 官网吗？ 域名已经想好了：pcl.fun

**Reply by Ad-closeNN** [reply]
https://github.com/Hex-Dragon/PCL2/discussions/1930#:~:text=%E5%90%8C%E6%AD%A5%E9%9A%90%E8%97%8F%E4%B8%BB%E9%A2%98%E3%80%81-,%E5%AE%98%E7%BD%91,-%E7%AD%89

![image](https://github.com/user-attachments/assets/520408d9-24fe-40dc-9f59-f472632b447e)

**Reply by AkarinLiu** [reply]
可以注册一个域名跳转至这个项目的 GitHub 主页

**Reply by Deep-Dark-Forest** [reply]
> 可以注册一个域名跳转至这个项目的 GitHub 主页

域名费用谁来承担呢？
而且就算有域名了，这样做也没啥必要，因为这个 GitHub 仓库链接在 B 站上到处都是，没必要用这个……

**Comment by Ad-closeNN** [comment]
目前应该能使用 afdian.com 来作为发布的临时解决方案

**Reply by AkarinLiu** [reply]
建议考虑使用 GitHub Pages 和 VuePress 搭建一个官网以做备用。

**Reply by AkarinLiu** [reply]
比较担心爱发电的发展

**Reply by Ad-closeNN** [reply]
GitHub Pages 在国内访问不佳，效果不如  Gitee Pages

**Reply by AkarinLiu** [reply]
可以使用 Netlify 或 Cloudflare Pages 部署网站，与 GitHub 网站代码仓库关联就行。

**Comment by Pigeon0v0** [comment]
没必要用 NSIS 打包安装包啊...

PCL 是绿色软件，随用随走

**Comment by Deep-Dark-Forest** [comment]
GitHub 上的二进制运行文件已经有了，在 https://github.com/Hex-Dragon/PCL2/blob/main/%E6%9C%80%E6%96%B0%E6%AD%A3%E5%BC%8F%E7%89%88.zip

至于 Gitee，参考 #1616
